src/test/scala/uk/gov/hmrc/api/specs/ErrorValidation_InvalidPayloadParameters.scala

```diff
@@ -178,11 +182,12 @@ class ErrorValidation_InvalidPayloadParameters extends BaseSpec with BaseHelper
 
     Scenario("NICC_TC_N008: Verify the request with Date of Birth Year >= 16 receives error response 400") {
       val payload  = PayloadMapping.getOrElse("NICC_TC_N008", fail("NICC_TC_N008 not found"))
+      val dobUnder16 = LocalDate.now().minusDays(1).minusYears(16).toString
```

Comment:
Is there a better name for this variable? Just from reading it, it is not clear to me what it is for.